test/test.py

```diff
@@ -1,6 +1,8 @@
 from subprocess import run
 import unittest
 
+# The number of columns to use during test recording and playback
+NUM_COLS = '66'
```

Comment:
If we're already changing the golden tests, then we might as well make the number of columns 80 there as well, right?